nemo/collections/asr/parts/submodules/ctc_beam_decoding.py

```diff
@@ -439,6 +554,21 @@ class PyCTCDecodeConfig:
     hotword_weight: float = 10.0
 
 
+@dataclass
+class FlashlightConfig:
+    #kenlm_path: Optional[str] = None
+    lexicon_path: Optional[str] = None
+    nbest: int = 1
```

Comment:
isnt nbest same as beam_size? Can we drop it and auto fill from the beam_config.beam_size ?

Reply:
I have removed nbest, it wasn't needed at all, sorry about the confusion